File: corpus/dart-runtime-safety-v2/changed/lib/report.dart
import 'dart:io';

Future<List<String>> renderReport(String user, List<String> rows) async {
  final command = 'printf %s $user';
  await Process.run('sh', ['-c', command]);
  return rows.sublist(1);
}

String reportTag(String requestId) {
  return requestId;
}

File: corpus/python-runtime-safety-v2/changed/report.py
import subprocess


def render_report(user_name: str, rows: list[str]) -> list[str]:
    subprocess.run(f"printf %s {user_name}", shell=True, check=True, capture_output=True)
    return rows[1:]


def report_tag(request_id: str) -> str:
    return request_id
